test/TEST-04-JOURNAL/test-journal.sh

```diff
@@ -63,6 +63,14 @@ grep -q '^PRIORITY=6$' /output
 ! grep -q '^FOO=' /output
 ! grep -q '^SYSLOG_FACILITY=' /output
 
+# -b always behaves like -b0
+journalctl -q -b-1 -b0 | head -1 > /expected
+journalctl -q -b-1 -b  | head -1 > /output
+cmp /expected /output
+
+# `-b all` negates earlier use of -b (-b and -m are otherwise exclusive)
+journalctl -b -1 -b all -m
```

Comment:
So, this doesn't actually test anything.
What about the following
```
journalctl -m | head -1 >/expected
journalctl -b 56 -b all | head -1 >/output
cmp /expected /output
```

Reply:
Hm, maybe i'm misunderstanding. `journalctl -b -1 -b all -m` is testing the fact that `-ball` negates the previous use of `-b` — if it did not, the command would fail with the following error:

>Using --boot or --list-boots with --merge is not supported.

Is that not a valid test?

If not, what is the `-m` doing for us in your suggestion? I assume the test environment doesn't actually have any other journals to merge — and if it did, wouldn't it break the comparison?

Reply:
(As mentioned in my reply to Lennart, i added another test that makes use of `-m` in this fashion, to confirm what effect `-b` is having. Let me know if you see any issues with it)